docs/content/configuration/index.md

```diff
@@ -221,12 +221,14 @@ If you are using a distributed cache such as memcached, you can include the conf
 |--------|-----------|-------|
 |`druid.cache.type`|`local`, `memcached`|The type of cache to use for queries.|`local`|
 |`druid.cache.unCacheable`|All druid query types|All query types to not cache.|["groupBy", "select"]|
+|`druid.cache.useCache`|Whether to use cache for getting query results.|false|
+|`druid.cache.populateCache`|Whether to populate cache.|false|
```

Comment:
thanks, yeah that is true. I copy/pasted the lines from "druid.cache.unCacheable" which is also wrong. let me update all of them.